openshift-knative-operator/cmd/operator/kodata/knative-eventing/0.23.0/1-eventing-crds.yaml

```diff
@@ -201,6 +245,176 @@ spec:
         - name: Reason
           type: string
           jsonPath: ".status.conditions[?(@.type==\"Ready\")].reason"
+    - !!merge <<: *version
+        name: v1alpha2
+        served: false
+        storage: false
+        schema:
+          openAPIV3Schema: &openAPIV3Schema
+            type: object
```

Comment:
I get:

```
error: error parsing 1-eventing-crds.yaml: error converting YAML to JSON: yaml: line 248: did not find expected key
```

Reply:
I'll try again, but those files were generated from the patch

Reply:
It's working on my machine. Which yq version do you have?